NodeJS/src/RestClient.ts

```diff
@@ -400,4 +459,18 @@ export class RestClient {
         return -error.response.status
       })
   }
+
+  private appendQueryParam(
+    query: string,
+    paramName: string,
+    paramValue: string | number
+  ) {
+    if (!query) {
+      query = '?'
+    } else {
+      query += '&'
+    }
+    query += paramName + '=' + paramValue
+    return query
+  }
```

Comment:
Nope nope, modifying a parameter is a super smell. Please create a new local variable. 

I recommend:

```
const connectorCharacter = !query ? '?' : '&'
const result = `${connectorCharacter}${paramName}=${paramValue}`
return result
```

Reply:
ok

Reply:
done